jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy

```diff
@@ -27,7 +27,7 @@ pipeline {
             name: 'CORTX_IMAGE'
         )
         choice (
-            choices: ['sanity','regression','audit_log','cli_framework','cmd_framework','conf_store','consul_service','iem_framework','kv_store','message_bus','support_bundle','validator'], 
```

Comment:
Please validate that cortx-py-utils PR pipeline is working with changes

Reply:
cortx-py-utils working fine
https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-cortx-py-utils/2947/